templates/bake/Template/login.twig

```diff
@@ -6,14 +6,15 @@
 ?>
 <?php $this->extend('/layout/TwitterBootstrap/signin'); ?>
 
-<?= $this->Form->create(${{ singularVar }}, ['class' => 'form-signin text-center']) ?>
+<?= $this->Form->create(${{ singularVar }}, ['class' => 'form-signin']) ?>
 <?= $this->Html->image('BootstrapUI.baked-with-cakephp.svg', ['class' => 'mb-4', 'width' => '250']) ?>
 <h1 class="h3 mb-3 font-weight-normal"><?= __('Please sign in') ?></h1>
-<?= $this->Form->label('email', __('Email address'), ['class' => 'sr-only']) ?>
+<?php if (isset($this->Flash)) echo $this->Flash->render(); ?>
```

Comment:
Helpers can be auto loaded and many users might not explicitly include them in helpers list, so I don't think the `isset()` check is a good idea.

Reply:
Yeah, I was copypasta-ing it from the default layout, without bothering to check if there actually was a reason why it was written like that.